test/testenv/secretutil.go

```diff
@@ -109,7 +107,8 @@ func ModifySecretObject(deployment *Deployment, data map[string][]byte, ns strin
 //ModifySecretKey Modifies the specific key in secret object
 func ModifySecretKey(deployment *Deployment, ns string, key string, value string) bool {
```

Comment:
Add a log to print which secret is being updatec

Reply:
added